e2e_samples/fabric_dataops_sample/infrastructure/terraform/modules/keyvault/main.tf

```diff
@@ -0,0 +1,26 @@
+#trivy:ignore:AVD-AZU-0016
+resource "azurerm_key_vault" "keyvault" {
+  name                       = var.keyvault_name
+  location                   = var.location
+  resource_group_name        = var.resource_group_name
+  tenant_id                  = var.tenant_id
+  sku_name                   = "standard"
+  purge_protection_enabled   = var.purge_protection_enabled
+  soft_delete_retention_days = var.soft_delete_retention_days
+  enable_rbac_authorization  = var.enable_rbac_authorization
+  tags                       = var.tags
+
+  access_policy {
+    tenant_id = var.tenant_id
+    object_id = var.object_id
+
+    secret_permissions = [
```

Comment:
Is there a reason why are we using the Classic Access Policies for KeyVault instead of the RBAC permissions? @promisinganuj

Reply:
@hassant , thanks for pointing this out.
This was an initial version which we are fixing as part of https://github.com/Azure-Samples/modern-data-warehouse-dataops/pull/862. We shall merge the PR 862 today so that it is updated.